index.html

```diff
@@ -0,0 +1,173 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="./style/normalize.css">
+    <link rel="stylesheet" href="./style/style.css" />
+    <title>coogle popup</title>
+    <link rel="apple-touch-icon" sizes="57x57" href="./favicons/apple-icon-57x57.png" />
+    <link rel="apple-touch-icon" sizes="60x60" href="./favicons/apple-icon-60x60.png" />
+    <link rel="apple-touch-icon" sizes="72x72" href="./favicons/apple-icon-72x72.png" />
+    <link rel="apple-touch-icon" sizes="76x76" href="./favicons/apple-icon-76x76.png" />
+    <link rel="apple-touch-icon" sizes="114x114" href="./favicons/apple-icon-114x114.png" />
+    <link rel="apple-touch-icon" sizes="120x120" href="./favicons/apple-icon-120x120.png" />
+    <link rel="apple-touch-icon" sizes="144x144" href="./favicons/apple-icon-144x144.png" />
+    <link rel="apple-touch-icon" sizes="152x152" href="./favicons/apple-icon-152x152.png" />
+    <link rel="apple-touch-icon" sizes="180x180" href="./favicons/apple-icon-180x180.png" />
+    <link rel="icon" type="image/png" sizes="192x192" href="./favicons/android-icon-192x192.png" />
+    <link rel="icon" type="image/png" sizes="32x32" href="./favicons/favicon-32x32.png" />
+    <link rel="icon" type="image/png" sizes="96x96" href="./favicons/favicon-96x96.png" />
+    <link rel="icon" type="image/png" sizes="16x16" href="./favicons/favicon-16x16.png" />
+    <link rel="manifest" href="./manifest.json" />
+    <meta name="msapplication-TileColor" content="#ffffff" />
+    <meta name="msapplication-TileImage" content="./favicons/ms-icon-144x144.png" />
+    <meta name="theme-color" content="#ffffff" />
+</head>
+
+<body>
+    <div class="container">
+        <div class="nav">
+            <div class="nav__element text">
+                <a href="#" class="nav__link">gmail</a>
+            </div>
+            <div class="nav__element text">
+                <a href="#" class="nav__link">images</a>
+            </div>
+            <div class="nav__element">
+                <div class="apps"></div>
+            </div>
+
+            <!-- popup -->
+
+            <input type="checkbox" class="nav__element pop-check" name="apps" />
+            <!-- <div class="outline"></div> -->
```

Comment:
Remove commented lines of code

Reply:
@Shizopo Thank you, the commented line is deleted.